bin/ci-run

```diff
@@ -27,6 +27,11 @@ echo "* Running all rspec specs"
 echo "* ******************************************************"
 bundle exec rspec --format=progress
 
+echo "* ******************************************************"
+echo "* Running spellr"
+echo "* ******************************************************"
+bundle exec spellr
```

Comment:
🐘 I think it's probably better to have this _before_ our heavier checks (probably even as the first check) because I assume its a lot faster and it'd be better to have your CI fail on spelling before rather than after it runs the 20m+ test suite...